pkg/scheduler/placement/provided_rule_test.go

```diff
@@ -119,7 +119,8 @@ partitions:
 	// unqualified queue with parent rule that exists directly in hierarchy
 	appInfo = newApplication("app1", "default", "testchild", user, tags, nil, "")
 	queue, err = pr.placeApplication(appInfo, queueFunc)
-	if queue != "root.testparent.testchild" || err != nil {
```

Comment:
Could you please rewrite it by assert? For example:
```go
	assert.NilError(t, err)
	assert.Equal(t, "root.testparent.testchild", queue)
```

Reply:
I'm sorry for not updating. I'll update right away.